omnisend/includes/SDK/V1/class-contact.php

```diff
@@ -36,9 +36,10 @@ class Contact {
 	private $email_consent = null;
 	private $phone_consent = null;
 
-	private $email_opt_in_source = null;
-	private $phone_opt_in_source = null;
-
+	private $email_opt_in_source     = null;
+	private $email_opt_out_source    = false;
```

Comment:
instead of `$email_opt_out_source` I would suggesting having string type for `$email_status` with valid values 
subscribed, nonsubscribed and unsubscribed. This should be validated before sending it to the API

Reply:
Changed to $email_status and $phone_status